src/index.js.flow

```diff
@@ -0,0 +1,13 @@
+/* flow strict */
+
+declare module 'auto-check-element' {
+  declare export default class AutoCheckElement extends HTMLElement {
+    get src(): string;
+    set src(value: string): void;
+    get csrf(): string;
+    set csrf(value: string): void;
+    get required(): boolean;
+    set required(required: boolean): void;
+    get input(): ?HTMLInputElement;
+  }
+}
```

Comment:
Can you separate the flow definitions into another pull request so that can merge? Coupling it to code changes means it can't merge until those pass code review.